app/helpers/react_helper.rb

```diff
@@ -29,6 +30,12 @@ def translations(locale)
     I18n.backend.translations.slice(:en, locale.to_sym)
   end
 
+  def search_page_alert(alert)
+    return if alert.text.blank? && alert.title.blank?
```

Comment:
What do you think about moving  this code from Line 15 `if affiliate.alert.present?` in here like
```ruby
return if !alert || alert.text.blank? && alert.title.blank?
```